graphql-client/src/Data/GraphQL/Error.hs

```diff
@@ -30,13 +33,35 @@ data GraphQLError = GraphQLError
   }
   deriving (Show, Eq, Generic, ToJSON, FromJSON)
 
+displayGraphQLError :: String -> String -> GraphQLError -> String
+displayGraphQLError indentFirst indentRest exception =
+  indentFirst
+    <> "At "
+    <> joinWithCommas (maybe [] (map displayErrorLoc) (locations exception))
```

Comment:
An empty list would make the message a bit weird. Can we maybe show `???` if there's no location (either Nothing or an empty list in `locations`)?